wcomponents-test-lib/src/test/java/com/github/bordertech/wcomponents/test/components/WCheckBoxFieldUI.java

```diff
@@ -0,0 +1,31 @@
+package com.github.bordertech.wcomponents.test.components;
+
+import com.github.bordertech.wcomponents.WApplication;
+import com.github.bordertech.wcomponents.WCheckBox;
+import com.github.bordertech.wcomponents.WFieldLayout;
+import com.github.bordertech.wcomponents.WMessages;
+import com.github.bordertech.wcomponents.WPanel;
+
+import static com.github.bordertech.wcomponents.WFieldLayout.LAYOUT_FLAT;
+
+public class WCheckBoxFieldUI extends WApplication {
+
+	private WMessages messages = new WMessages();
+	private WPanel panel = new WPanel();
+
+	public WCheckBoxFieldUI() {
+
+		panel.add(messages);
+		WFieldLayout fieldLayout = new WFieldLayout(LAYOUT_FLAT);
+		fieldLayout.setTitle("this is a bunch of fields");
+
+		//WTextField
+		WCheckBox wCheckBox = new WCheckBox();
+		fieldLayout.addField("this is a checkbox", wCheckBox);
+
+		panel.add(fieldLayout);
+		add(panel);
+	}
```

Comment:
Is this really sufficient to test your WCheckBoxWebElement? One check box is sufficient if it is exposed through a getter so the tests can set properties on it.